testcafe/tests/helpers/visualreview-utils.js

```diff
@@ -29,6 +29,7 @@ export class VisualReviewTestcafe extends VisualReview {
 
   //@param { jsonObject } masK { x, y, width, height } : coordonate for mask. order is the same in gimp rectangle selection
   async setMaksCoordonnates(maskCoordonates) {
+    this.resetMask
```

Comment:
Why is this lin here? It's also not calling the function, so I think we can remove it :)